### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by tdeekens**
...before releasing.

### CODE REVIEW

File: application-templates/starter/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "react": "16.8.6",
     "react-apollo": "2.5.5",
     "react-dom": "16.8.6",
-    "react-intl": "2.8.0",
+    "react-intl": "2.9.0",
```
Comment (tdeekens) [inline application-templates/starter/package.json]: Will update with it in the mc too to have one version.

File: package.json [hunk 2]
```diff
@@ -83,19 +83,19 @@
...
+    "lint-staged": "8.1.7",
+    "prettier": "1.17.1",
+    "puppeteer": "1.16.0",
     "rcfile": "1.0.3",
     "read-pkg-up": "6.0.0",
-    "rollup": "1.11.3",
+    "rollup": "1.12.1",
     "rollup-plugin-babel": "4.3.2",
-    "rollup-plugin-commonjs": "9.3.4",
+    "rollup-plugin-commonjs": "10.0.0",
```
Comment (tdeekens) [inline package.json]: These now support new side-effect features.

Comment (montezume) [inline package.json]: I was looking at this, I think the "breaking" change is it requires rollup the new rollup, which you updated as well 👍

File: packages/mc-scripts/package.json [hunk 3]
```diff
@@ -55,12 +55,12 @@
     "postcss-reporter": "6.0.1",
     "postcss-safe-parser": "4.0.1",
     "promise": "8.0.3",
-    "react-dev-utils": "7.0.3",
+    "react-dev-utils": "9.0.1",
```
Comment (tdeekens) [inline packages/mc-scripts/package.json]: The breaking change here is the method signature of `createCompiler` to accept an object. I fixed that in `mc-scripts`.

File: package.json (hunk 2 above)
Comment (tdeekens) [reply package.json]: Yes. That's how I understood it too when reading through their PRs.

**Pull request merged**